### DISCUSSION TITLE
Changing web ui to https(443) from http(80) process

### DISCUSSION THREAD

**Discussion by Scbri** [aqualinkd/AqualinkD#325] [Q&A]
Just wanted to ask about config changes need to update UI work on port 443 for https support on raspberry pi? is it just a matter of enabling port 443 and changing www config to port 443 from port 80? getting error in HA due to HA serving on https?

**Comment by sfeakes** [comment]
AqualinkD does not support https, only http.  It’s not designed to be used over the internet, only lan, so no need for https.   HA is fine with http, what are you trying to do?

**Reply by Scbri** [reply]
i had added a view on a dashboard using the webpage card and was getting an error saying frames can't use http when HA was serving on https. i was testing an old ipad with the companion app to use as a wall controller.

**Reply by Scbri** [reply]
![image](https://github.com/user-attachments/assets/06cee7e1-3a41-4134-8ee5-d75987d3779d)

**Comment by sfeakes** [comment]
You could always connect to HA with http (locally), or put a https proxy in front of AqualinkD.

**Reply by Scbri** [reply]
y i'm thinking the same thing. probably add one of the proxy addons with let's encrypt cert for self signed certs
